save-backend/build.gradle.kts

```diff
@@ -49,16 +52,34 @@ tasks.register<Exec>("cleanupDbAndStorage") {
     dependsOn(":liquibaseDropAll")
     val profile = properties["save.profile"] as String?
 
+    val userHome = System.getProperty("user.home")
+    val isWindows = System.getProperty("os.name").startsWith("Windows")
+
     val storagePath = when (profile) {
-        "win" -> "${System.getProperty("user.home")}/.save-cloud/cnb/files"
+        "win" -> "$userHome/.save-cloud/cnb/files"
         "mac" -> "/Users/Shared/.save-cloud/cnb/files"
-        else -> "/home/cnb/files"
-    }
+        else -> when {
+            isWindows -> "$userHome/.save-cloud/cnb/files"
+            else -> "/home/cnb/files"
+        }
+    }.let(Paths::get)
 
-    val args = if (profile != "win") {
+    /*
+     * No idea why we should rely on running external commands in order to
+     * delete a directory, since this can be done in a platform-independent way.
```

Comment:
I'm not sure, but maybe it was not obvious how to wrap deletion into a Gradle tasks correctly? But definitely feel free to replace it.

Reply:
Thanks, will rewrite it in a short while.